style.css

```diff
@@ -104,37 +105,44 @@ color: #172B4D;
   gap: 8px;
   order: 3;
 }
-.twitter :hover{
+
+.twitter :hover {
   color: #505f79;
   opacity: 0.5;
 }
-.linkedin :hover{
+
+.linkedin :hover {
   color: #505f79;
   opacity: 0.5;
 }
-.github :hover{
+
+.github :hover {
   color: #505f79;
   opacity: 0.5;
 }
-.medium :hover{
+
+.medium :hover {
   color: #505f79;
   opacity: 0.5;
 }
-.angellist :hover{
+
+.angellist :hover {
   color: #505f79;
   opacity: 0.5;
 }
-/*-------------cards--------------*/
+
+/* -------------cards-------------- */
 
 .cards-container {
-    display: grid;
-    grid-template-columns: 1fr;
-    box-sizing: border-box;
-    height: 174.25rem;
-    width: 23.44rem;
-    margin-top: -1137px;
-  }
-  .cards {
+  display: grid;
+  grid-template-columns: 1fr;
+  box-sizing: border-box;
+  height: 174.25rem;
+  width: 23.44rem;
+  margin-top: -1137px;
+}
+
+.cards {
```

Comment:
- [ ] Weldone with this, @malembi :wink: but the use of hard-coded or absolute units is affecting your design, not making it really appealing enough to show the great efforts you have invested in it. This is what it looks like for all your cards:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/219012966-74130e80-9d9d-4052-8a25-a22e39740f9c.png)|![image](https://user-images.githubusercontent.com/99715640/219013344-4ff4e6b2-b542-4d82-9bfd-0f6a12d28d55.png)|

- [ ] Kindly make your design consistent with the Figma by checking your use of explicit `height` and `width` properties.

Reply:
okay